core/context.js

```diff
@@ -120,19 +123,16 @@ class TelegrafContext {
   }
 
   get state () {
-    if (!this.contextState) {
-      this.contextState = {}
-    }
-    return this.contextState
+    return (this.contextState = this.contextState || {})
```

Comment:
Not great for code readability

Reply:
Why? I think, it is enough clear.
Really not great is https://github.com/telegraf/telegraf/blob/develop/markup.js#L144-L167

Reply:
Because of this: https://eslint.org/docs/rules/no-return-assign

Reply:
![image](https://user-images.githubusercontent.com/5930429/33542395-f2501db8-d8e3-11e7-9fb8-1f5a98396a0e.png)
Eslint not shows error, when brackets are present!

Reply:
Yes, but it's just cheating.

Reply:
k man. It's your app )